SwiftWisdom/Core/Controls/VideoPlayer.swift

```diff
@@ -57,6 +57,12 @@ public struct Video {
     }
 }
 
+extension Video {
+    static public func == (lhs: Video, rhs: Video) -> Bool {
+        return (lhs.fileName == rhs.fileName) && (lhs.type == rhs.type) && (lhs.url == rhs.url)
```

Comment:
Since the `Bundle` is one of the defining properties, don't we need to check that for equality too?